mne/preprocessing/stim.py

```diff
@@ -67,3 +79,108 @@ def eliminate_stim_artifact(raw, events, event_id, tmin=-0.005,
         elif mode == 'window':
             raw[picks, first_samp:last_samp] = data * window[np.newaxis, :]
     return raw
+
+
+def fix_stim_artifact(inst, events=None, event_id=None, tmin=-0.2,
+                      tmax=-0.1, mode='linear'):
```

Comment:
@agramfort I think tmax=0.01 is too small. For tutorials, It has sfreq=150. For this freq, it will make error because (tmax - tmin)*sfreq is less than 4. How about tmax=0.05? It would not make error except sfreq is less than 80, which is scarcely case.